.travis.yml

```diff
@@ -4,8 +4,6 @@ language: php
 
 matrix:
   include:
-    - php: 5.6
-    - php: 7.0
```

Comment:
@sanyokerem Can you remove this commit, and rebase your branch with master? Then we can merge it.

Reply:
Enqueue requires php7.1+ so could not be rebased with master.

Reply:
Ok, wait for https://github.com/LongRunning/LongRunning/pull/30 to be merged

Reply:
#30 is merged can you rebase this, we can then merge it and release a new version.